src/rfxgen.c

```diff
@@ -1172,842 +1103,6 @@ static void ProcessCommandLine(int argc, char *argv[])
 }
 #endif      // PLATFORM_DESKTOP
 
-//--------------------------------------------------------------------------------------------
-// Load/Save/Export functions
-//--------------------------------------------------------------------------------------------
-
-// Reset wave parameters
-static void ResetWaveParams(WaveParams *params)
-{
-    // NOTE: Random seed is set to a random value
-    params->randSeed = GetRandomValue(0x1, 0xFFFE);
-    srand(params->randSeed);
-
-    // Wave type
-    params->waveTypeValue = 0;
-
-    // Wave envelope params
-    params->attackTimeValue = 0.0f;
-    params->sustainTimeValue = 0.3f;
-    params->sustainPunchValue = 0.0f;
-    params->decayTimeValue = 0.4f;
-
-    // Frequency params
-    params->startFrequencyValue = 0.3f;
-    params->minFrequencyValue = 0.0f;
-    params->slideValue = 0.0f;
-    params->deltaSlideValue = 0.0f;
-    params->vibratoDepthValue = 0.0f;
-    params->vibratoSpeedValue = 0.0f;
-    //params->vibratoPhaseDelay = 0.0f;
-
-    // Tone change params
-    params->changeAmountValue = 0.0f;
-    params->changeSpeedValue = 0.0f;
-
-    // Square wave params
-    params->squareDutyValue = 0.0f;
-    params->dutySweepValue = 0.0f;
-
-    // Repeat params
-    params->repeatSpeedValue = 0.0f;
-
-    // Phaser params
-    params->phaserOffsetValue = 0.0f;
-    params->phaserSweepValue = 0.0f;
-
-    // Filter params
-    params->lpfCutoffValue = 1.0f;
-    params->lpfCutoffSweepValue = 0.0f;
-    params->lpfResonanceValue = 0.0f;
-    params->hpfCutoffValue = 0.0f;
-    params->hpfCutoffSweepValue = 0.0f;
-}
-
-// Generates new wave from wave parameters
-// NOTE: By default wave is generated as 44100Hz, 32bit float, mono
-static Wave GenerateWave(WaveParams params)
```

Comment:
Avoid raylib type: `Wave` can be replaced by custom `rfxWave` and mapping done on user side.